tests/exporters/onnx/test_exporters_onnx_cli.py

```diff
@@ -28,6 +28,11 @@
     from optimum.exporters.tasks import TasksManager
 
 import subprocess
+import sys
+from pathlib import Path
+
+
+sys.path.insert(0, Path(__file__).parent.parent.as_posix())
```

Comment:
Please no, i think it is bad practice. Just put everything under the same folder if necessary.